content/en/docs/concepts/containers/runtime-class.md

```diff
@@ -144,11 +140,9 @@ See CRI-O's [config documentation](https://raw.githubusercontent.com/cri-o/cri-o
 
 {{< feature-state for_k8s_version="v1.16" state="beta" >}}
 
-As of Kubernetes v1.16, RuntimeClass includes support for heterogeneous clusters through its
-`scheduling` fields. Through the use of these fields, you can ensure that pods running with this
-RuntimeClass are scheduled to nodes that support it. To use the scheduling support, you must have
-the [RuntimeClass admission controller](/docs/reference/access-authn-authz/admission-controllers/#runtimeclass)
-enabled (the default, as of 1.16).
+If the nodes in your cluster support different sets of features, you can set the `scheduling`
+field for a RuntimeClass, then use that hint to ensure that pods running with that particular
+RuntimeClass are scheduled to nodes that support it.
```

Comment:
This paragraph now needs some tweaks for clarification. For example, what does "sets of features" mean? How many feature can we cover using the `scheduling` field and/or RuntimeClass? What does "set the `scheduling` field for a RuntimeClass" mean?

Reply:
This paragraph is based on sftim's suggestion. @sftim Any idea?

Reply:
IMO this is good enough (but with room to improve further). If / when this PR merges, we can try to rally help from SIG Scheduling, or indeed from anyone interested, to improve the documentation further.

@tengqm please feel free to suggest alternative wording. If we can't easily think up an alternative then to me that's evidence that it's better to merge the thing as-is, provided it is not making the page obviously worse.

Reply:
Well ... my current understanding is base on the API spec for the `Scheduling` definition. The RuntimeClass was mainly devised for different container runtimes, however, the design was generalized to handle many scenarios. Let me try put it here for language tweaks. 

> By specifying the `scheduling` field for a RuntimeClass, you can set constraints to
> ensure that Pods running with this RuntimeClass are scheduled to nodes that support it. 
> If `scheduling` is not set, this RuntimeClass is assumed to be supported by all nodes.

I'll take back my comments about the "scheduling on runtimeClass" part because that is already explained in the paragraphs below.